tests/dummy/app/templates/multi-card.hbs

```diff
@@ -0,0 +1,97 @@
+{{md-text
+  text="
+## Multi card
+
+The `nypr-multi-card` component can be used to create simple links between cards by using the yielded `panel` component. At minimum, each `panel` takes a `to` attribute, which points to another panel by its index in the DOM tree.
+
+### Basic Usage
+Use the exposed `button` property on the yielded `panel` to place the button that will switch to another panel.
+```htmlbars
+{{#nypr-multi-card as |m|}}
+  
+  {{#m.panel to=1 as |panel|}}
+    Here's panel 0. It's the default because it's first in the DOM.<br />
+    {{panel.button text=\"Show me the next panel\"}}
+  {{/m.panel}}
+  
+  {{#m.panel to=0 as |panel|}}
+    Here's panel 1. You switch to it using the button from panel 0.<br />
+    {{#panel.button}}You can also open the button if you want more complicated markup{{/panel.button}}
+  {{/m.panel}}
+  
+{{/nypr-multi-card}}
+```
```

Comment:
I like where your head's at. I think we can improve upon this by moving the `to` parameter to the yielded `nypr-card/button` component. agree not a current use case but something we can easily extend to support.